metricbeat/module/ceph/osd_df/osd_df.go

```diff
@@ -0,0 +1,58 @@
+package osd_df
+
+import (
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/libbeat/common/cfgwarn"
+	"github.com/elastic/beats/metricbeat/helper"
+	"github.com/elastic/beats/metricbeat/mb"
+	"github.com/elastic/beats/metricbeat/mb/parse"
+)
+
+const (
+	defaultScheme = "http"
+	defaultPath   = "/api/v0.1/osd/df"
+)
+
+var (
+	hostParser = parse.URLHostParserBuilder{
+		DefaultScheme: defaultScheme,
+		DefaultPath:   defaultPath,
+	}.Build()
+)
+
+func init() {
+	if err := mb.Registry.AddMetricSet("ceph", "osd_df", New, hostParser); err != nil {
+		panic(err)
+	}
+}
+
+type MetricSet struct {
+	mb.BaseMetricSet
+	*helper.HTTP
+}
+
+func New(base mb.BaseMetricSet) (mb.MetricSet, error) {
+	cfgwarn.Beta("The ceph osd_df metricset is beta")
```

Comment:
I suggest we make this first experimental (as we do with all new metricsets first)

Reply:
Ok, I will change it to experimental metricbet.